packages/ember-views/tests/views/checkbox_test.js

```diff
@@ -139,3 +139,36 @@ QUnit.test('checking the checkbox updates the value', function() {
   equal(!!checkboxComponent.$().prop('checked'), false, 'after clicking a checkbox, the checked property changed');
   equal(get(checkboxComponent, 'checked'), false, 'changing the checkbox causes the view\'s value to get updated');
 });
+
+QUnit.test('sanity check for indeterminate', function() {
+  /* global jQuery */
+  var checkbox = jQuery('<input type="checkbox">').appendTo('#qunit-fixture');
+
+  var changeEventFired = false;
+  checkbox.on('change', function(e) { changeEventFired = true; });
+
+  // The presence of indeterminite: true on the checkbox causes phantomjs to not dispatch the
+  // change event.  Comment this out and re-run this test.
+  checkbox.prop('indeterminate', true);
```

Comment:
@rwjblue - I got the phantomjs tests running, a few console logs and a few hours later I traced it down to this problem.

It seems setting `indeterminite` to `true` on a checkbox in phantomJS stops the `change` event from being fired when it is clicked on.

This bug is not present in any "real" browsers, and I'm not sure how to fix this assumption in the Ember code or the test suite to be able to test this in PhantomJS.  Again, why am I spending 2-3 hours to fix and track down bugs in a browser that no user actually uses?